### PR TITLE
Bug 1772049: Add VMs to Project Dashboard

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/public/components/dashboard/project-dashboard/inventory-card.tsx [hunk 1]
```diff
@@ -43,33 +46,67 @@ const ProjectInventoryItem = withDashboardResources(
...
     mapper,
     useAbbr,
+    additionalResources = [],
   }: ProjectInventoryItemProps) => {
     React.useEffect(() => {
       if (projectName) {
         const resource = createFirehoseResource(model, projectName);
         watchK8sResource(resource);
-        return () => stopWatchK8sResource(resource);
+        if (additionalResources.length) {
```
Comment (mareklibra) [inline frontend/public/components/dashboard/project-dashboard/inventory-card.tsx]: is this `if` needed? It exists as long as `additionalResources` is an array, even empty

Comment (rawagner) [reply frontend/public/components/dashboard/project-dashboard/inventory-card.tsx]: not needed really, removed

**Pull request merged**